lib/optimizely.rb

```diff
@@ -39,7 +39,8 @@ class Project
     attr_reader :event_dispatcher
     attr_reader :logger
 
-    def initialize(datafile, event_dispatcher = nil, logger = nil, error_handler = nil, skip_json_validation = false, user_profile_service = nil)
+    def initialize(datafile, event_dispatcher = nil, logger = nil, error_handler = nil, skip_json_validation = false,
```

Comment:
nit. Looks awkward with just 1 parameter on the next line. Perhaps do an equal split.